pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -216,10 +215,10 @@ func (c *Client) OperatorVersionsInstalled(operatorName, namespace string) ([]st
 	if err != nil {
 		return nil, err
 	}
-	existingVersions := []string{}
+	existingVersions := make([]string, 0)
```

Comment:
nit: I believe that syntax on the left is preferable unless you define a custom size.

Reply:
IntelliJ complained about it, referring to https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

I changed it back for now